R/file.R

```diff
@@ -92,9 +95,10 @@ file_copy <- function(path, new_path, overwrite = FALSE) {
 
 #' Change ownership or group of a file
 #' @template fs
-#' @param user_id The user id of the new owner, the R process must be privlaged to change
-#'   this.
-#' @param group_id The group id of the new owner
+#' @param user_id The user id of the new owner, specified as a numeric ID or
+#'   name. The R process must be privileged to change this.
```

Comment:
BTW I don't know what "R process must be privileged" means

Reply:
It essentially means you have to run the R process as a super user (either as root or su) to change it (https://en.wikipedia.org/wiki/Privilege_(computing)#Unix)

Reply:
So this means that these functions are of relatively little use then (generally)?  I think that means we should just do the minimum here and wait for user requests before going deeper.